packages/appium/docs/en/intro/index.md

```diff
@@ -2,4 +2,200 @@
+people using any popular programming language.
+
+Luckily, the fact that Appium rode in on Selenium's coattails meant that we had a solution to this
+problem from day one. You see, the WebDriver specification is actually an HTTP-based protocol,
+meaning it is designed to be used over a network rather than within the memory of a single program.
+While there are some slight drawbacks in terms of performance and complexity to this model, the
+fact that we have a 'client-server' architecture is extremely beneficial.
+
+One of the main benefits is that it allows the automation implementer (the thing doing the
+automation, in this case the 'server') to be completely distinct from the automation runner (the
+thing defining what automation should be done, in what steps, etc..., in this case the 'client').
+Basically, all the "hard stuff" (actually figuring out how to make automation happen on a given
+platform) can be handled in one place by the server, and "thin" client libraries can be written in
+any programming language which simply encode HTTP requests to the server in language-appropriate
+way. It's possible, in other words, to bring basic Appium / WebDriver capabilities to a new
+programming language in less than a day's work, simply by coding up a basic HTTP client in that
+language.
+
+There are a couple important takeaways here for you, the Appium user:
+
+- Appium is an *HTTP server*. It must run as a process on some computer for as long as you want to
+  be able to use it for automation. It must be accessible on the network to whichever computer you
+  want to use to run the automation from (whether that is the same machine or one across the
+  world).
+- Unless you want to write raw HTTP calls or use cURL, using Appium for automation involves the use
+  of an [Appium Client](clients.md) in the language of your choice. The goal of each of these
+  clients is to encapsulate the WebDriver protocol so that rather than worrying about the protocol
+  itself, you can work with objects and methods that feel idiomatic for your language.
+- The Appium server and the Appium client do *not* need to be running on the same computer. You
+  simply need to be able to send HTTP requests from the client to the server over some network.
+  This greatly facilitates the use of cloud providers for Appium, since they can host the Appium
+  server and any related drivers and devices, and all you need to do is point your client script to
+  their secure endpoints.
+
+And of course, none of this is about "testing" per se, purely about the use of Appium and its
+client libraries for automation purposes. If you want to do automation for the purpose of
+"testing", you'll likely want to enlist the help of test runners, test frameworks, and the like,
+none of which need be related to Appium; one of the benefits of Appium's "universal accessibility"
+is that it plays well with whatever set of tools you find most beneficial for your situation.
```

Comment:
in the general sense (not this document), we need to do more outreach/promotion around the testing ecosystem.  because a) most people using appium are writing tests, and b) they are not going to have a great experience if the only tool at their disposal is an appium client.  the mistakes selenium has made come along for the ride on their coattails...

Reply:
agreed